.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,19 @@
+
+## Changes
+<!-- Bullet list of key changes -->
+
+## Testing
+<!-- What testing was done? -->
+- [ ] `make fmt` passes (no formatting drift)
+- [ ] `make test` passes
+- [ ] Unit tests added or updated (required for features and bug fixes)
+- [ ] Manual testing performed (if applicable)
+
+## Checklist
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)